Geometry/HcalCommonData/python/cmsExtendedGeometry2021XML_cfi.py

```diff
@@ -234,8 +234,8 @@
         'Geometry/HcalCommonData/data/hcalouteralgo/v1/hcalouteralgo.xml',
         'Geometry/HcalCommonData/data/hcalforwardalgo.xml',
         'Geometry/HcalCommonData/data/average/hcalforwardmaterial.xml',
-        'Geometry/HcalCommonData/data/hcalSimNumbering/2021/v2/hcalSimNumbering.xml',
-        'Geometry/HcalCommonData/data/hcalRecNumbering/2021/v1/hcalRecNumbering.xml',
+        'Geometry/HcalCommonData/data/hcalSimNumbering/2021/v1/hcalSimNumbering.xml',
+        'Geometry/HcalCommonData/data/hcalRecNumbering/2021/hcalRecNumbering.xml',
```

Comment:
The PR description does not explain why you are appling these fixes (?) to the code just merged with #42463

Reply:
I am keeping the old file for comparison. The fixed one should be in CMSCommonData